### DISCUSSION TITLE
zk is not found after installation

### DISCUSSION THREAD

**Discussion by S-N-O-R-L-A-X** [openbuildxyz/mina-zkapp-bootcamp#227] [Q&A]
![image](https://github.com/user-attachments/assets/9a132892-3dda-4434-b30f-bdad68794b04)
![image](https://github.com/user-attachments/assets/69953049-325c-4d01-a1e1-57ba661e42a1)

**Comment by MartinYeung5** [comment]
my setting is that:
![螢幕擷取畫面 2024-11-26 211906](https://github.com/user-attachments/assets/cce98509-93a9-46a2-990e-106d99a6bb57)

**Reply by S-N-O-R-L-A-X** [reply]
Interesting...

**Reply by S-N-O-R-L-A-X** [reply]
But I don't think it is the version problem.

**Comment by MartinYeung5** [comment]
by the way, better to use WSL in windows
